test/services-test/multi_mem_volatile.txt.in

```diff
@@ -0,0 +1,11 @@
+-n=1
+--model=memory_server
+--cisinterface=rpc
+--memserverinterface=rpc
+--metaserverinterface=rpc
+--cisrpcaddr=127.0.0.1:8787
+--memservers=0:{memory_type:volatile,fam_path:/dev/shm/nsou_vol,rpc_interface:127.0.0.1,rpc_port:8795,libfabric_port:7500,if_device:eth0}
```

Comment:
Do not give your username in this file, We may fix it in run_test.py as part of some other PR.

Reply:
Given for testing. will change it